.github/workflows/test-application.yaml

```diff
@@ -33,27 +65,31 @@ jobs:
             - name: Install and configure PHP
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: '7.4'
+                  php-version: ${{ matrix.php-version }}
                   extensions: ctype, iconv, mysql
                   coverage: pcov
-                  tools: composer
+                  tools: ${{ matrix.tools }}
 
             - name: Install composer dependencies
               uses: ramsey/composer-install@v1
               with:
-                  dependency-versions: 'highest'
+                  dependency-versions: ${{ matrix.dependency-versions }}
+                  composer-options: ${{ matrix.composer-options }}
 
             - name: Bootstrap test environment
               run: composer bootstrap-test-environment
 
             - name: Lint code
+              if: ${{ matrix.lint }}
               run: composer lint
 
             - name: Execute unit test cases
               run: composer test -- Tests/Unit --coverage-php var/coverage.php --coverage-html coverage-report --log-junit var/junit.xml
 
             - name: Execute functional test cases
+              if: ${{ matrix.functional }}
```

Comment:
There would be some refractoring needed to create snapshot for sulu 2.1 and sulu 2.2 so I did currently disable testing functional tests on php 7.3 as they currently all fails.

/cc @nnatter

Reply:
think i would just increase the min sulu version for now 🙂

Reply:
Done!